### ISSUE TITLE
<Multiselect/>  - cannot see tag text upon typing

### ISSUE THREAD

**Issue by miroshSa** [wix/wix-style-react#3257]
# 🐛 Bug Report

### 🏗 Relevant Components

`<Multiselect/>`

...

### 😯 Current Behavior

You cannot see tag text upon typing
...

### 🤔 Expected Behavior

You can see tag text upon typing

...

### 👣 Steps to Reproduce

  - **WSR version**: `6.12.0`

### 👀 Severity

- Critical

**Comment by ipanasenko** [comment]
This commit seems to introduce the issue: https://github.com/wix/wix-style-react/commit/7079e7b08047cb61f9dc537c1fed4528562f78ad

**Comment by shlomitc** [comment]
Looks like the issue happens only for the `uncontrolled` mode of the component.

broken:
```jsx
class MyComp extends React.Component {
  render() {
    return <MultiSelect/>;
  }
```

works:
```jsx
class MyComp extends React.Component {
  constructor() {
    this.state = {
      value: ''
    }
  }
  render() {
    return <MultiSelect value={this.state.value} onChange={e => this.setState({value: e.target.value})}/>;
  }
}
```

**Comment by mosheKerbel** [comment]
Yes, looking at the docs of autoSizeInput
it doesn't support it, need to figure out a solution
https://github.com/JedWatson/react-input-autosize#uncontrolled-input

**Comment by ipanasenko** [comment]
We've just released a fix yesterday for this in our project, we are now controlling it. Do you think it might be better to deprecate uncontrolled behavior altogether?

**Comment by mosheKerbel** [comment]
@shlomitc 
i see a few options here, WDYT?
1. fork AutosizeInput and modify the code to support uncontrolled input.
    It is currently calculating the input width by the injected value. Need to give it a prop of isControlled.
    When isControlled=false, take the value from the input dom.
2. when we are in uncontrolled mode do not use autoSizeInput, fallback to regular input.
3. deprecate uncontrolled behaviour.

**Comment by shlomitc** [comment]
for now - let's make a quick fix to add this behavior as an opt-in, just so we won't need to revert it.
I prefer keeping the uncontrolled behavior, so let's try to solve it

**Comment by shlomitc** [comment]
fixed in this PR https://github.com/wix/wix-style-react/pull/3277 and released in 6.15.0
